de-de/python-de.html.markdown

```diff
@@ -119,10 +119,10 @@ False or True #=> True
 "{name} will {food} essen".format(name="Bob", food="Lasagne")
```

Comment:
I'd prefer if we focus heavily on f-strings, like in the English version of the python3 page

Reply:
I have deleted the `.format`-Version. 

I hope you meant this because I was not 100% sure about it.